Jenkinsfile-rf-pipeline

```diff
@@ -0,0 +1,324 @@
+env.ghprbGhRepository = env.ghprbGhRepository ?: 'CentOS-PaaS-SIG/ci-pipeline'
+env.ghprbActualCommit = env.ghprbActualCommit ?: 'master'
+
+library identifier: "ci-pipeline@${env.ghprbActualCommit}",
+        retriever: modernSCM([$class: 'GitSCMSource',
+                              remote: "https://github.com/${env.ghprbGhRepository}"])
```

Comment:
we need to add the refspec here maybe?

Reply:
`traits: [[$class: 'RefSpecsSCMSourceTrait', templates: [[value: '+refs/pull/*:refs/remotes/origin/pr/*']]]]]`